modules/add_remove.js

```diff
@@ -1 +1,25 @@
-﻿
\ No newline at end of file
+export const booklist = document.querySelector('.books-list');
+export const title = document.querySelector('#Title');
+export const author = document.querySelector('#Author');
+
+export const addBooks = (booksArr) => {
+  booklist.innerHTML = '';
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < booksArr.length; i++) {
+    booklist.innerHTML += `
+      <div class="title-author"> 
+        <p class="Title">"${booksArr[i].title}" by  ${booksArr[i].author}</p>
+        <button class="button remove-books" id = "${i}">Remove</button>
+      </div>
+     `;
+    title.value = '';
+    author.value = '';
+    title.focus();
+  }
+};
+
+export const deleteBook = (el) => {
+  if (el.classList.contains('remove-books')) {
+    el.parentElement.remove();
+  }
+};
```

Comment:
- [ ] Deleted book still re-display on the browser after page reloading. Kindly look into that and fix.

[app-del.webm](https://user-images.githubusercontent.com/26597432/206988343-5f98bd5a-095b-4d7e-a6d4-0af1b1e74823.webm)